Rulebook 3.0 (#113)
International-Data-Spaces-Association/IDSA-Rulebook: 003_WhatIsADataspace.md

# What is a Data Space?
The ISO/IEC 20151 Standard "Information technology — Cloud computing and distributed platforms — Dataspace concepts and characteristics" defines Data Spaces as:

> ***"environment enabling trusted data sharing between participating parties, based on an agreed governance framework, along with an agreed set of policies, semantic models, standardized protocols, processes, and facilitating services"***

Reviewer: Which version of the ISO/IEC 20151 standard is this description from? In the version that I bought in September 2025 the description of the term "dataspace" is a little bit different:

> "governance framework and supporting services to build trustworthiness and enable data sharing through an agreed set of policies, semantic models, protocols and processes"

Author: it's from the latest version that was used for the approval process by ISO SC38 members. There was a change in the definition to align with the Cen/Cenelec definition.

Reviewer: In that case, my version is the outdated one.

- **agreed governance framework** - The Data Space Governance Framework is the defined set of technical policies, business rules, and regulations that participants in a data space have to adhere to. It is the core agreement between all parties, which defines a data space. The Data Space Governance Authority is mandated to maintain and enforce the Data Space Governance Framework. The functional requirements section on [Data Space Governance Authorities (DSGAs)](...) explains the functions, responsibilities and interactions of such a framework in more detail. 
- **agreed set of policies** - Within a data space there are multiple layers of rules which ensure the creation of trust between two participants. Those rules are expressed as policies, specifically data space membership policies, access policies, contract policies and usage policies. More details on policies can be found in the functional requirements section on [Policies](...).

> - **semantic models** - As the policies used in a data space are very flexible and pretty much any arbitrary rule can be expressed it is important to create a model of what policies can be used and what they mean. Sharing common semantic models between the participating parties greatly enhances the interoperability within the data space. Exchanging semantic models of the data shared further improves its use and enables value generation. [Semantic Models](...) are explained as well in great detail in their respective section of the IDSA Rulebook.

Reviewer 2: ```suggestion
- **semantic models** - Sharing common semantic models on (1) the data that is to be shared and (2) the data that describes the data space itself (e.g. policies, participants, processes) between the participating parties greatly enhances the interoperability within the data space. Exchanging semantic models of the data shared further improves its use and enables value generation. [Semantic Models](...) are explained as well in great detail in their respective section of the IDSA Rulebook.
```

> - **standardised protocols** - The insanly fast growth and adoption of the WWW would not have been possible without common protocols, such as TCP/IP and HTTP, which enable software from many different vendors, servers at many different providers and browsers on different phones and computers to seamlessly work toghether. Data spaces rely on a similar set of basic protocols that are the foundation for all technical communication: The [Dataspace Protocol (DSP)](https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/) and the [Decentralized Claims Protocol (DCP)](https://eclipse-dataspace-dcp.github.io/decentralized-claims-protocol/). 

Reviewer 2: ```suggestion
- **standardised protocols** - The incredibly rapid growth and widespread adoption of the WWW would not have been possible without common protocols, such as TCP/IP and HTTP, which enable software from many different vendors, servers at many different providers and browsers on different phones and computers to seamlessly work together. Data spaces are built on a similar set of foundational protocols that enable technical communication between participants, namely the [Dataspace Protocol (DSP)](https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/) and the [Decentralized Claims Protocol (DCP)](https://eclipse-dataspace-dcp.github.io/decentralized-claims-protocol/). 
```

- **processes** - Different communities will have different processes for onboarding and managing their community members. Having those processes clearly defined, governed and well managed enables smooth collaboration between participating parties.

> - **facilitating services** - Many data spaces will cooperate with external providers to provide services which make the operation of the data space and the sharing of data in that data space easier. This can include onboarding services, auditing, marketplaces, and many more. These services will be described throught the IDSA Rulebook, but obviously there are many more than can be described here as this is the area where innovation will create new business models, new governance models and services that can't be anticipated at this point in time.

Reviewer 2: ```suggestion
- **facilitating services** - Many data spaces will cooperate with external service providers to support their operations and make data sharing easier and more secure. These services may include onboarding, auditing, marketplaces, and others. The IDSA Rulebook outlines key service categories and governance principles but does not provide an exhaustive list. As data spaces evolve, new business models, new governance models and services will naturally emerge.
```

## Autonomy and Agency

> The most important aspect of data spaces is the **autonomy** and **agency** of a participant, commonly referred to as "digital sovereignty". Autonomy refers to the capability to make decisions of when to share which data with whom and under what conditions. Agency is the capability to perform the data sharing. This is essentially only possible if the participant controls all technical elements required to participate in a data space. Any external service that is mandatory (that is, required by the DSGA or by law/regulation) in the negotiation of a data sharing agreement or in the execution of such an agreement—for example, where a central or federated catalog is the only means to discover data or a marketplace is the only service that enables contract negotiation—reduces autonomy and agency.

Reviewer 2: ```suggestion
The most important aspect of data spaces is the **autonomy** and **agency** of a participant, commonly referred to as "digital sovereignty". Autonomy refers to the ability to decide whether to share specific data, with whom, and under which conditions. Agency refers to the ability to execute those data sharing decisions in practice. This is only fully possible if the participant controls all technical elements required to participate in a data space. Any external service that is mandatory (that is, required by the DSGA or by law/regulation) in the negotiation of a data sharing agreement or in the execution of such an agreement — for example, where a central or federated catalog is the only means to discover data or a marketplace is the only service that enables contract negotiation — reduces autonomy and agency.
```

However, full autonomy and agency come at a price. The participating party needs to have the ability and technical means to control all technical, business and legal elements of participating in the data space and negotiating contracts. For many participants this will be too expensive or too cumbersome in relation to the value of the data shared. In those cases data intermediaries can be used which will perform data space access functions and perform the sharing or use of data. As each function that is provided by a service provider impairs the participants digital sovereignty it needs to be carefully weighted which functions can and should be handed over to a service provider. 

> Especially for smaller organisations that need to share data in data spaces that mostly serve regulatory purposes it might be a good option to not participate directly but use the services of an aggregator / portal provider who then shares their data on their behalf to the data space.

Reviewer 2: ```suggestion
Especially for smaller organisations participating in data spaces that primarily serve regulatory purposes, it may be preferable not to connect directly, but instead to use an aggregator or portal provider that shares data with the data space on their behalf.
```

The IDSA Rulebook will explain the functionality, mechanics and processes of a data space from the perspective of a fully autonomous participant with full agency. Where applicable additional explanation will be provided how intermediaries, aggregators and other service providers can be leveraged to trade digital sovereignty for ease of access and operation.

## Communities

> Data spaces are data ecosystems build on the concept of communities of trusted participants. There are many potential perspectives of what a community of participants is. They can be based on a jurisdictional community (e.g. all participants are from the same country), industry group (e.g. all participants are energy providers, or health care providers), based on a common customers (e.g. a data space for all suppliers of a multi-national company), or any private community, for example founded to join forces on one very specific business use case.

Reviewer 2: ```suggestion
Data spaces are data ecosystems built on the concept of communities of trusted participants. There are many potential perspectives of what a community of participants can be. They can be based on a jurisdictional community (e.g. all participants are from the same country), industry group (e.g. all participants are energy providers, or health care providers), based on a common customers (e.g. a data space for all suppliers of a multi-national company), or any private community, for example founded to join forces on one very specific business use case.
```

![Data Spaces are based on Communities of Organizations](./media/communities.png)

> Data spaces an overlap or be aligned hierarchically. E.g. the community of all European Healthcare Organizations is organized in the European Healthcare Data Space (EHDS), which can be segmented into smaller Healthcare data spaces by country, while at the same time being segmented into separate data spaces by industry (e.g. pharma, hospitals, medical device manufacturers,...) and again be also segmented by a specific use case (e.g. Cancer Research Data, Disease Tracking, ...).

Reviewer: Should this be:

> Data spaces can overlap or be aligned hierarchically.

> Data spaces can overlap or be aligned hierarchically. E.g. the community of all European Healthcare Organisations is organized in the European Healthcare Data Space (EHDS), which can be segmented into smaller Healthcare data spaces by country, while at the same time being segmented into separate data spaces by industry (e.g. pharma, hospitals, medical device manufacturers,...) and again be also segmented by a specific use case (e.g. Cancer Research Data, Disease Tracking, ...).

Reviewer 2: ```suggestion
Data spaces can overlap or be aligned hierarchically. As an example, the community of all European Healthcare Organisations is organised in the European Healthcare Data Space (EHDS), which can be segmented into smaller healthcare data spaces by country, while at the same time being segmented into separate data spaces by industry (e.g. pharma, hospitals, medical device manufacturers) and again be also segmented by a specific use case (e.g. cancer research, disease tracking).
```

> No matter which community a organization wants to participate in they need to have the technical means (e.g. operating a data space connector that understand the two foundational protocols, knowledge of the respective semantic models) and it needs to have the necessary credentials that proof that it is a member of this specific community. 

Reviewer: This long sentence could be clarified a bit, for example:

> No matter which community an organization wants to participate in, it needs to have the technical means (e.g., operating a data space connector that understands the two foundational protocols and knowledge of the relevant semantic models). It also needs to have the necessary credentials that prove it is a member of that specific community.

> No matter which community a organisation wants to participate in they need to have the technical means (e.g. operating a data space connector that understand the two foundational protocols, knowledge of the respective semantic models). It needs to have the necessary credentials that proof that it is a member of this specific community. 

Reviewer 2: ```suggestion
No matter which community an organisation wants to participate in, it needs to have the technical means for participation. This includes, for example, operating a data space connector that supports the required foundational protocols and having knowledge of the relevant semantic models. In addition, the organisation needs to possess the appropriate credentials that prove its membership in the respective community.
```

> Some communities will require significant effort to join (e.g. becoming part of a not-for-profit association, signing legal documents, etc...) while others might be relative lightweight and offer a low friction onboarding process (e.g. sharing open data for scientific research).

Reviewer: "relative" should be "relatively"

> Some communities will require significant effort to join (e.g. becoming part of a not-for-profit association, signing legal documents, etc...) while others might be relatively lightweight and offer a low friction onboarding process (e.g. sharing open data for scientific research).

Reviewer 2: ```suggestion
Some communities will require significant effort to join (e.g. becoming part of a not-for-profit association, signing legal documents). Others may offer a more lightweight, low-friction onboarding process, for example when sharing open data for scientific research.
```

### International Cross-Border Data Sharing

> Special attention is necessary when the community consists of organisations operating under different jurisdictions as this might create legal conflicts in the policies required to share data. While some might be relatively easy to resolve (e.g. data sharing within the member states of the European Union) other will require more investments to resolve conflicts and clarify ambigous rules (e.g. sharing data between a Chinese and a German company as part of a supply chain data space).

Reviewer 2: ```suggestion
Special attention is necessary when the community consists of organisations operating under different jurisdictions as this might create legal conflicts in the policies required to share data. While some might be relatively easy to resolve (e.g. data sharing within the member states of the European Union), others will require more investments to resolve conflicts and clarify ambiguous rules (e.g. sharing data between a Chinese and a German company as part of a supply chain data space).
```

Based on those two protocols any variety of software agents can be built. Standalone, single server deployments, Connectors-as-a-Service at a cloud provider, software agents as a feature of a large enterprise software, AI Agents with Dataspace capabilities, and many more. The concept is open to future innovation and interoperability is guaranteed through the protocol specifications which embody the durable core concept of data spaces.

## Business Models
Data spaces are agnostic to the business model(s) of the data ecosystem being enabled by them. It is possible to build data spaces that contain only free, open or altruistic data, others that are designed to share regulatory mandated data or others that are build around the idea of creating a marketplace for data. Any business model can be built on data spaces and with data space technology. Even platforms and aggregators can use data space technology to implement the need for a decentralized mechanism to negotiate resource sharing contracts.

> It will also be very common to see data spaces that support than one business model at a given time. e.g. data that needs to be shared for free within the supply chain due to legal regulation can also be offered in a data marketplace to other companies at a price.

Reviewer 2: ```suggestion
Data spaces can also support more than one business model at a given time. e.g. same data that needs to be shared for free within the supply chain due to legal regulation can also be offered in a data marketplace to other companies at a price.
```

Data spaces are designed to foster future innovation in business models, data use, value creation and monetization.

> Same as the internet and WWW enabled the creation of new business models and monetization strategy and the value this created for the global economy as well as for each individual user. Data spaces will enable trusted data sharing, which will lead to new insights, new business models, new opportunities and the creation of value for all participants! 

Reviewer: Consider updating this for better readability, for example:

> Similarly, the Internet and the World Wide Web enabled the creation of new business models and monetization strategies, generating significant value for the global economy as well as for individual users. In the same way, data spaces will enable trusted data sharing, leading to new insights, new business models, new opportunities, and value creation for all participants.

> Similarly, the Internet and the World Wide Web enabled the creation of new business models and monetization strategies, generating significant value for the global economy as well as for individual users. In the same way, data spaces will enable trusted data sharing, leading to new insights, new business models, new opportunities, and value creation for all participants. 

Reviewer 2: ```suggestion
The Internet and the World Wide Web have enabled new business models and monetisation strategies, creating significant value for the global economy and for individual users. Similarly, data spaces enable trusted data sharing, unlocking new insights, business models, opportunities, and value for all participants.
```
